optuna/study.py

```diff
@@ -848,6 +848,55 @@ def objective(trial):
 
         self._storage.create_new_trial(self._study_id, template_trial=trial)
 
+    @experimental("2.5.0")
+    def add_trials(self, trials: Sequence[FrozenTrial]) -> "Study":
```

Comment:
I think the minimum requirements of `trials` argument is `Iterable` since it is just iterated by for-loop. But the `Sequence` is an interface to ensure `Reversible` and `Collection` (`Sized`, `Iterable`, and `Container`).
https://docs.python.org/3/library/typing.html#typing.Sequence

How about using `Iterable[FrozenTrial]` here?

Reply:
Thank you. I changed it.